### PR TITLE
FOIA-326: Updates documentation for Tooltip, Modal, Checkbox Widget, …

### PR DESCRIPTION
**Pull request by elm45**
**Updates documentation for:**
- Tooltip
- Modal
- Table

**Adds documentation for:**
- Checkbox Widget
- Form Submit

**Also adds CSS documentation.**

### CODE REVIEW

File: docs/annual-report-data-app.md [hunk 1]
```diff
@@ -64,15 +66,22 @@ import FoiaTooltip from '../components/foia_tooltip';
...
+ * foia_report_form_section_three
 
 
 #### Modal
 
 Props include:
-* `triggerText` which is used inside the button that triggers the modal.
-* `ariaLabel` which is used as the aria label for the modal.
-* `modalContent` which is the content that goes inside the modal.
+* `triggerText` The visible text in the button that triggers the modal.
```
Comment (bemarlan) [inline docs/annual-report-data-app.md]: Can you add colons here after the prop name?

File: docs/annual-report-data-app.md [hunk 2]
```diff
@@ -255,9 +270,66 @@ Example use:
...
+```
+
+Used in:
+ * foia_report_form_section_three
+
+
+#### Foia Report Data Submit
+
+Props include
+  `selectedDataTypes`: An array of the currently selected DataTypes.
```
Comment (bemarlan) [inline docs/annual-report-data-app.md]: Can you make these bullets? Same style as lines 284-287.

File: docs/annual-report-data-app.md (hunk 1 above)
Comment (elm45) [reply docs/annual-report-data-app.md]: Resolved by lastest commit.

File: docs/annual-report-data-app.md (hunk 2 above)
Comment (elm45) [reply docs/annual-report-data-app.md]: Resolved by latest commit.

**Pull request merged**